homeworks_js/h.w-27/javascript.js

```diff
@@ -0,0 +1,112 @@
+//1. Написать функцию, которая принимает 2 числа и возвращает -1, если первое меньше, чем второе; 
+// 1 – если первое больше, чем второе; и 0 – если числа равны.
+function getNumber(number1, number2) {
+    if (number1 < number2) {
+        return -1; 
+    } else if (number1 > number2) {
+        return 1;     
+    } else if (number1 == number2) {
+        return 0;    
+    } else 
+        return "uknown"
+    }
+ getNumber(7, 6)
+//2. Написать функцию, которая вычисляет факториал переданного ей числа
+function factorial(number) {
+    if (number == 0) {
+      return 1;
+    } else {
+      return number * factorial(number - 1);
+    }
+  }
+  console.log(factorial(4))
+
+//3. Написать функцию, которая принимает три отдельные цифры и превращает их в одно число. 
+// Например: цифры 1, 4, 9 превратятся в число 149.
+
+
+//4. Написать функцию, которая принимает длину и ширину прямоугольника и вычисляет его площадь. 
+// Если в функцию передали 1 параметр, то она вычисляет площадь квадрата.
+function rectangleValue(a, b) {
+      rectangleArea = a * b;
```

Comment:
you can omit creation of unused  variable here just return  a * b; but you don't handle case when user enter only one parametr